cds_migrator_kit/handlers.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2024 CERN.
+#
+# cds-migrator-kit is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS Migrator Records logging handler."""
+
+import logging
+
+cli_logger = logging.getLogger("migrator")
+documents_logger = logging.getLogger("documents_logger")
+items_logger = logging.getLogger("items_logger")
+
+
+def migration_exception_handler(exc, output, key, value, rectype=None, **kwargs):
+    """Migration exception handling - log to files.
+
+    :param exc: exception
+    :param output: generated output version
+    :param key: MARC field ID
+    :param value: MARC field value
+    :return:
+    """
+    logger = logging.getLogger(f"{rectype}s_logger")
+    cli_logger.error(
+        "#RECID: #{0} - {1}  MARC FIELD: *{2}*, input value: {3}, -> {4}, ".format(
```

Comment:
General comment for the future: @jrcastro2 will add structured JSON logging to `invenio-jobs`, and it would be nice to start using it a bit everywhere when we have want to log long running jobs output, so that we have similar fields/structure.